apps/api/src/domain/models/audio-visual/audio-item/repositories/arango-audio-item-query-repository.spec.ts

```diff
@@ -412,4 +414,71 @@ describe(`ArangoAudioItemQueryRepository`, () => {
             expect(missingLineItems).toEqual([]);
         });
     });
+
+    describe(`translate line item`, () => {
+        const originalLanguageCode = LanguageCode.Chilcotin;
+
+        const translationLanguageCode = LanguageCode.English;
+
+        const translationText = 'this is how to translate what was said to English';
+
+        const inPointMilliseconds = 100;
+
+        const outPointMilliseconds = 200;
+
+        const participant = buildTestInstance(TranscriptParticipant);
+
+        const targetLineItem = buildTestInstance(TranscriptItem, {
+            inPointMilliseconds,
+            outPointMilliseconds,
+            speakerInitials: participant.initials,
+            text: buildMultilingualTextWithSingleItem('original text', originalLanguageCode),
+        });
+
+        const existingTranscript = buildTestInstance(Transcript, {
+            participants: [participant],
+            items: [targetLineItem],
+        });
+
+        const targetAudioItem = buildTestInstance(EventSourcedAudioItemViewModel, {
+            transcript: existingTranscript,
+        });
+
+        beforeEach(async () => {
+            await testQueryRepository.create(targetAudioItem);
+        });
+
+        it('should translate the line item', async () => {
+            await testQueryRepository.translateLineItem(targetAudioItem.id, {
+                inPointMilliseconds,
+                outPointMilliseconds,
+                text: translationText,
+                languageCode: translationLanguageCode,
+                speakerInitials: participant.initials,
```

Comment:
Is this necessary? The in and out point are sufficient to uniquely identify the line item.

Reply:
done